### PR TITLE
Wifi 13796 remove haproxy, rrm and kafka brokers

### PR DESCRIPTION
**Pull request by carsten989**
WIFI-13865 - Remove RRM
Remove kafka brokers - these cause issues for current components (components are less likely to come up)

### CODE REVIEW

File: chart/environment-values/values.base.insecure.yaml [hunk 1]
```diff
@@ -106,4 +92,4 @@ kafka:
   broker:
     persistence:
       size: 20Gi
-    replicaCount: 2
+    replicaCount: 0
```
Comment (i-chvets) [inline chart/environment-values/values.base.insecure.yaml:95]: Should we just remove `broker` section?

File: chart/environment-values/values.custom.tpl.yaml [hunk 2]
```diff
@@ -38,43 +40,48 @@ owfms:
...
         external-dns.alpha.kubernetes.io/hostname: prov-${NAMESPACE}.${DOMAIN}
+        service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ${CERTIFICATE_ARN}
   configProperties:
     openwifi.system.uri.public: https://prov-${NAMESPACE}.${DOMAIN}:16005
     openwifi.system.uri.private: $INTERNAL_RESTAPI_ENDPOINT_SCHEMA://owprov-owprov:17005
     openwifi.system.uri.ui: https://provui-${NAMESPACE}.${DOMAIN}
     iptocountry.ipinfo.token: "${IPTOCOUNTRY_IPINFO_TOKEN}"
   public_env_variables:
     OWSEC: sec-${NAMESPACE}.${DOMAIN}:16001
+owgwui:
```
Comment (i-chvets) [inline chart/environment-values/values.custom.tpl.yaml:59]: Why `owgwui` added here?

Comment (i-chvets) [reply chart/environment-values/values.custom.tpl.yaml:59]: I see it is being moved.

File: chart/environment-values/values.base.insecure.yaml (hunk 1 above)
Comment (carsten989) [reply chart/environment-values/values.base.insecure.yaml:95]: no, let's leave it, It should be an option in HA setups.

File: chart/environment-values/values.custom.tpl.yaml (hunk 2 above)
Comment (carsten989) [reply chart/environment-values/values.custom.tpl.yaml:59]: just moved it from a little up in this file.

File: chart/environment-values/values.base.insecure.yaml (hunk 1 above)
Comment (i-chvets) [reply chart/environment-values/values.base.insecure.yaml:95]: Need to put a comment somewhere about it.

**Review by i-chvets — APPROVED** [review]
Approved.

**Pull request merged**